### ISSUE TITLE
(GSOC) Add Start A Hunt in the APP

### ISSUE THREAD

**Issue by ankit2001** [Bugheist/Flutter#54]
[Figma Link](https://www.figma.com/file/s0xuxeU6O2guoWEfA9OElZ/Bugheist-Full-Design?node-id=0%3A1)

<img width="258" alt="start_a_bughunt" src="https://user-images.githubusercontent.com/44541855/122679472-a5e8c980-d208-11eb-8f35-f123065e69e0.png">

**Comment by fredfalcon** [comment]
Would like to see a link to it even though you’d have to login.

**Comment by just-ary27** [comment]
> Would like to see a link to it even though you’d have to login.

Can we discuss this on slack rn?

**Comment by fredfalcon** [comment]
Yes… it would be a link in the left nav… than redirect to login if not logged in.

**Comment by fredfalcon** [comment]
![DD4FD129-9E0E-45B1-95B5-778A10EE6E9C](https://user-images.githubusercontent.com/7475382/180658755-f48ac206-2d27-4452-8976-086f6a45a43c.jpeg)

**Comment by just-ary27** [comment]
Currently we have this, the user can switch between the report and start bug hunt screen, I made it so as these actions are very similar to each other.
![image](https://user-images.githubusercontent.com/76696648/180659010-ec3f4914-4248-4c67-bd26-d92a972a9329.png)

**Comment by fredfalcon** [comment]
would like it to be a red button on the left nav and show even if a user is not logged in.

**Comment by fredfalcon** [comment]
Just need a link For un logged in users.  I usually don’t login so I haven’t seen this page yet

**Comment by letsintegreat** [comment]
Hey! What is the exact requirement here? For the guest users - a red button on the left nav which says "Start a bug hunt" and clicking on it redirects to the login page. Anything other than that?

**Comment by just-ary27** [comment]
Add a snackbar to suggest that you need to login as well

**Comment by letsintegreat** [comment]
@just-ary27 Alright!
